src/overlays/actors/ovl_Bg_Mori_Hashira4/z_bg_mori_hashira4.c

```diff
@@ -20,27 +27,142 @@ const ActorInit Bg_Mori_Hashira4_InitVars = {
     (ActorFunc)BgMoriHashira4_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Mori_Hashira4/func_808A3060.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Mori_Hashira4/func_808A3068.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 700, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_VEC3F_DIV1000(scale, 1000, ICHAIN_STOP),
+};
+
+extern ColHeader D_06001AF8;
+extern ColHeader D_060089E0;
+
+static Gfx* displayLists[] = { 0x06001300, 0x06008840 };
+
+static s16 unkTimer; //seems to be unused
```

Comment:
2 small things here: static/local variables are prefixed with `s` and comments are required to have a space before their content (this part should be handled by clang-format so I suggest running it again on this file just in case).
```suggestion
static Gfx* sDisplayLists[] = { 0x06001300, 0x06008840 };

static s16 sUnkTimer; // seems to be unused
```